python/lsst/daf/butler/registries/sql.py

```diff
@@ -870,6 +870,7 @@ def queryDatasets(self, datasetType: Any, *,
                     dimensions=dimensions,
                     dataId=standardizedDataId,
                     where=where,
+                    bind=bind,
```

Comment:
Oops 😄 

I admit I had looked for some examples of `bind` recently and didn't find many.